src/fides/api/models/privacy_preference.py

```diff
@@ -42,7 +41,7 @@ class RequestOrigin(Enum):
 class ConsentMethod(Enum):
     button = "button"
     gpc = "gpc"
-    individual_notice = "api"
+    individual_notice = "individual_notice"
```

Comment:
Old copy past error - @allisonking are we even using this field? It's the "method" when saving consent preferences and I think it's subject to change with the new consent reporting work

Reply:
oh interesting, no we aren't using this field in the frontend, though maybe we should be? Is `individual_notice` meant to be when a user toggles different notices then clicks "Save", while "button" is for "accept all"/"reject all"?

Reply:
perfect! although just to be clear, we _are_ using `ConsentMethod` in the frontend and using `button` and `gpc`, but not `individual_notice`

Reply:
got it okay 👍